File: atcoder.jp/contests/abc285/tasks/abc285_a/main.go
package main

import (
	"fmt"
)

func YesNo(b bool) string {
	if b {
		return "Yes"
	} else {
		return "No"
	}
}

func main() {
	var a, b int
	fmt.Scanf("%d %d", &a, &b)

	ans := a == b/2

	fmt.Println(YesNo(ans))
}

File: atcoder.jp/contests/abc285/tasks/abc285_a/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];

const div = (x: number, y: number) => Math.floor(x / y);

const [a, b] = input().split(" ").map(Number);

const ans = a === div(b, 2);

console.log(ans ? "Yes" : "No");
